apps/examples/testcase/le_tc/compression/tc_lzma.c

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <crc32.h>
+#include <tinyara/lzma/LzmaLib.h>
+
+unsigned int propsSize = LZMA_PROPS_SIZE;
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: tc_2Kb
+ *
+ * Description:
+ *   Check and verify the checksum of a file of size 2KB after compression and decompression
+ *
+ * Returned Value:
+ *   None
+ ****************************************************************************/
+void tc_2Kb(void)
```

Comment:
```
static void tc_compression_lzma_2kb(void)
```

Reply:
TC naming rule is `<prefix>_<module name>_<test name>_<postfix>`.
prefix is `tc`, `utc` or `itc`.
postfix is `p` or `n`.

Reply:
Made the changes